week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -84,7 +88,14 @@ function getTransportModes() { }
 
   Hint: Use the corresponding array method to decide if an element is member of an array.
 */
-function isAccessibleByTransportMode() { }
+function isAccessibleByTransportMode(arraysOfTransport,transportMode) { 
+  if (arraysOfTransport.find(item => item ===transportMode) === transportMode){
+    return true
+  }
+  else{
+    return false
+  }
```

Comment:
In this case, using `includes` would give more compact code. Or also you can use some, but it doesn't need such an extra logic.